### DISCUSSION TITLE
Design the blogs page blog.html

### DISCUSSION THREAD

**Discussion by akshitagupta15june** [akshitagupta15june/PetMe#155] [General]
Till now i have one blog received from Dan Schenker-: https://docs.google.com/document/d/14W-uGQFLrQySVddv-jzwPRu2vCVYpkRahCgOB34wApY/edit?usp=sharing

he has contributed in our repo

A template that i liked very much is attached
just surfing google for templates but we have to create from scratch

![image](https://user-images.githubusercontent.com/57909583/193568174-b96608d0-10ec-46ca-b6f0-9ea578eafb40.png)

**Comment by akshitagupta15june** [comment]
multiple contributors can work on it so please comment if u want to be assigned
also i am creating a discussion for this issue so that all contributors can discuss how to proceed

**Reply by Anuran12** [reply]
It would be great and easy if we use Next Js. Because next js is SEO friendly. But then we have to change the whole codebase into next js 🥲

**Comment by piyushsahujtp** [comment]
I need to have work on this issue kindly provide more information regarding this if required a html file or css file for this type of blog along with the template . Now i'm providing my blog code link https://github.com/piyushsahujtp/blog.git
for docs file kindly check my docs file blog : https://docs.google.com/document/d/1IdXNG-51DgmNyuAVYDDwyRsj21gZdR-sk60vEBonjU4/edit?usp=sharing
